.circleci/config.yml

```
@@ -81,7 +81,7 @@ jobs:
   test_integration:
     <<: *defaults
     docker:
-      - image: circleci/node:8.16.1-browsers@sha256:e2f01766ecc1e133ca4873b7a0fa292da2d39e756783f6a7f90d4d5c2a59ccb7
+      - image: circleci/node:12.14.1-browsers
```

You could keep the sha256 just to be sure to always pull the same image:

```suggestion
      - image: circleci/node:12.14.1-browsers@sha256:e50f174a64b0cc2622bd9c3202ef2e1c3495ff281ed653da344350745ac606a2
```

---

Oh I didn't know it was for that. Thanks 👍